string type
geometry_msgs/Pose2D center
float64 size_x
float64 size_y